docs/version-history/feature-versions-23-x.rst

```diff
@@ -22,6 +22,10 @@ New Features:
   a job.
   :jira:`2510`
 
+- Added a new verb to ``htcondor credential``, ``listall``, which allows the
```

Comment:
```suggestion
- Added a new verb to :tool:`htcondor credential listall`, which allows the
```
FYI: the above works to create a reference to the `htcondor` tool manual

Reply:
Note you could also keep this similar to what you have as such:
:tool:`htcondor credential`, ``listall``

Reply:
The second form makes more sense to me; commit pushed.